### PR TITLE
Update extension service and deployment script

### PR DESCRIPTION
**Pull request by fpagnoux**
Connected to openfisca/openfisca-core#576
Depends on openfisca/openfisca-core#583

Should be merged and applied right before merging openfisca/openfisca-core#583 to avoid any service interruption.

### CODE REVIEW

File: auto-update-pip-packages/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 OpenFisca-France[api] >= 18.0.0, < 19.0
-OpenFisca-Core[tracker]
+OpenFisca-Tracker == 0.1.0
```
Comment (sandcha) [inline auto-update-pip-packages/requirements.txt]: Why not OpenFisca-Tracker 0.2.0 ? 🤔

File: fr.openfisca.org/api/deploy.sh [hunk 2]
```diff
@@ -2,13 +2,10 @@
 
 set -ex
 
-REQUIREMENTS="/home/openfisca/openfisca-ops/auto-update-pip-packages/requirements.txt"
-NEW_REQUIREMENTS="/home/openfisca/new-api/requirements.txt"
-
-sed -e 's/\[api\]//' "$REQUIREMENTS" > "$NEW_REQUIREMENTS"
+REQUIREMENTS="/home/openfisca/openfisca-ops/fr.openfisca.org/api/requirements.txt"
```
Comment (sandcha) [inline fr.openfisca.org/api/deploy.sh]: 👍

File: fr.openfisca.org/api/openfisca-web-api-new.service [hunk 3]
```diff
@@ -3,11 +3,8 @@ Description=OpenFisca Web API - New
 
 [Service]
 EnvironmentFile=/home/openfisca/secret/piwik_token
-Environment=COUNTRY_PACKAGE=openfisca_france
 Environment=SERVER_NAME=fr.openfisca.org/api/v18
-Environment=TRACKER_URL=https://stats.data.gouv.fr/piwik.php
-Environment=TRACKER_IDSITE=4
-ExecStart=/home/openfisca/virtualenvs/new-api/bin/gunicorn "openfisca_web_api_preview.app:create_app()" --workers 3 --bind localhost:6000
+ExecStart=/home/openfisca/virtualenvs/new-api/bin/openfisca serve --port 6000 --tracker-idsite 4 --tracker-url https://stats.data.gouv.fr/piwik.php --country_package openfisca_france --tracker-token $TRACKER_TOKEN
```
Comment (sandcha) [inline fr.openfisca.org/api/openfisca-web-api-new.service]: `--country-package` ?

File: fr.openfisca.org/api/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+OpenFisca-France >= 18.0.0, < 19.0
```
Comment (sandcha) [inline fr.openfisca.org/api/requirements.txt]: Why not >= 18.10 as in [update-api-cmd branch](https://github.com/openfisca/openfisca-france/tree/update-api-cmd) ?

File: auto-update-pip-packages/requirements.txt [hunk 5]
```diff
@@ -1,2 +1,2 @@
 OpenFisca-France[api] >= 18.0.0, < 19.0
```
Comment (sandcha) [inline auto-update-pip-packages/requirements.txt]: Why not >= 18.10 as in [update-api-cmd branch](https://github.com/openfisca/openfisca-france/tree/update-api-cmd) ?

Comment (fpagnoux) [reply auto-update-pip-packages/requirements.txt]: This is for the old API. The old API doesn't need 18.1.0

**Pull request merged**